app/controllers/public/schedule.js

```diff
@@ -9,4 +9,43 @@ export default class ScheduleController extends Controller {
   get isSchedulePublished() {
     return this.model.event.schedulePublishedOn.toISOString() !== moment(0).toISOString();
   }
+
+   hideSchedule = true;
+   hideTrack = false;
+   hideRooms = false;
+  @action
+  showSchedule() {
+    this.toggleProperty('hideSchedule');
+    this.toggleProperty('hideTrack');
+    this.toggleProperty('hideRooms');
+
+	  this.set('hideSchedule', true);
+	  this.set('hideTrack', false);
+	  this.set('hideRooms', false);
+
+  }
+
+  @action
+  showTrack() {
+    this.toggleProperty('hideSchedule');
+    this.toggleProperty('hideTrack');
+    this.toggleProperty('hideRooms');
+    
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6077057)